test/PR_test/integration_test/trace/test_logger.py

```diff
@@ -43,46 +42,44 @@ def test_on_begin_global_step(self):
         logger = Logger()
         logger.system = sample_system_object()
         logger.system.global_step = 2
-        self._test_print_msg(func=logger.on_begin, data=self.data, msg=self.on_begin_global_step_msg)
+        self._test_print_msg(func=logger.on_begin,
+                             data=self.data, msg=self.on_begin_global_step_msg)
 
     def test_on_begin_global_step_not_none(self):
         logger = Logger()
         logger.system = sample_system_object()
-        self._test_print_msg(func=logger.on_begin, data=self.data, msg=self.on_begin_msg)
+        self._test_print_msg(func=logger.on_begin,
+                             data=self.data, msg=self.on_begin_msg)
 
     def test_on_batch_end(self):
         logger = Logger()
         logger.system = sample_system_object()
         logger.system.global_step = 1
         logger.system.log_steps = 3
-        self._test_print_msg(func=logger.on_batch_end, data=self.data, msg=self.on_batch_end_msg)
+        self._test_print_msg(func=logger.on_batch_end,
+                             data=self.data, msg=self.on_batch_end_msg)
 
     def test_on_epoch_end_mode_train(self):
         logger = Logger()
         logger.system = sample_system_object()
         logger.system.global_step = 2
         logger.system.log_steps = 3
-        self._test_print_msg(func=logger.on_epoch_end, data=self.data, msg=self.on_epoch_end_train_msg)
-
-    def test_on_epoch_end_mode_eval(self):
```

Comment:
Removed this test case, since we will not print the "FastEstimator-Eval: step:" since we are printing the progress.

Reply:
why not update for the new print rather than just delete the test case?